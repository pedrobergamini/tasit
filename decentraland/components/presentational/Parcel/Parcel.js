import React from "react";
import { Image, StyleSheet } from "react-native";
import PropTypes from "prop-types";
import {
  responsiveWidth,
  responsiveHeight,
} from "react-native-responsive-dimensions";

export default function Parcel({ parcel }) {
  let { img } = parcel;

  return (
    <React.Fragment>
      <Image style={styles.landImage} source={{ uri: img }} />
    </React.Fragment>
  );
}

Parcel.propTypes = {
  parcel: PropTypes.object.isRequired,
};

const styles = StyleSheet.create({
  landImage: {
    height: responsiveHeight(20),
    width: responsiveWidth(95),
  },
});
