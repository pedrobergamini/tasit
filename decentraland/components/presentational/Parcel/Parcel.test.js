import React from "react";
import { shallow } from "enzyme";
import Parcel from "@presentational/Parcel";

describe("Parcel", () => {
  it("renders the component", async () => {
    const parcel = {
      id: 123,
      name: "Not found",
      img: "https://decentraland.org/images/logo-65f7b27caf.png",
    };
    expect(shallow(<Parcel parcel={parcel} />)).toMatchSnapshot();
  });
});
