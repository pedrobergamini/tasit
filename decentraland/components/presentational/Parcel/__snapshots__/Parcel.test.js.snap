// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parcel renders the component 1`] = `
<Fragment>
  <Component
    source={
      Object {
        "uri": "https://decentraland.org/images/logo-65f7b27caf.png",
      }
    }
    style={
      Object {
        "height": 266.8,
        "width": 712.5,
      }
    }
  />
</Fragment>
`;
